src/content/guides/tenant-cluster-labelling/index.md

```diff
@@ -18,6 +18,8 @@ Label keys and values are freely modifiable except labels with keys containing `
 Working with tenant cluster labels works likewise as working with Kubernetes labels.
 More information about Kubernetes Labels can be found in the [Kubernetes Labels and Selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) and our [cluster labels API documentation](/api/#tag/cluster-labels).
 
+Note: You can also [manage cluster labels directly through `happa`](/reference/web-interface/tenant-cluster-labelling/), our web user interface.
```

Comment:
I guess the other way around as well? So mentioning this from happa docs?

Reply:
It's already there